### ISSUE TITLE
Can't upgrade if MySql server is case-sensitive

### ISSUE THREAD

**Issue by Magneticdud** [i7MEDIA/mojoportal#89]
Hi, if, like me, the MySql server is hosted on a Linux server and is case sensitive, upgrading to a newer version of MojoPortal fails. Luckily the fix is simple:

in "www\Setup\applications\blog\SchemaUpgradeScripts\mysql\2.6.1.9.config"

replace the code with this:

```
DELETE FROM mp_ModuleDefinitionSettings WHERE SettingName = 'BlogShowAddFeedLinksSetting';
DELETE FROM mp_ModuleSettings WHERE SettingName = 'BlogShowAddFeedLinksSetting';
```

Maybe there are other instances that I missed because in the beginning I took the wrong approach of renaming the tables to lowercase

**Comment by Magneticdud** [comment]
did another upgrade, also this need to be fixed:

in "wwwroot\Setup\applications\mojoportal-core\SchemaUpgradeScripts\mysql\2.6.0.0.config"

`ALTER` TABLE `mp_modulesettings` 

needs to become

`ALTER` TABLE `mp_ModuleSettings`

**Comment by ElijahFowler** [comment]
Fixed in commits df247368d1dbee1e930edb41fab02287e003cc46, 0d824dfda5d2f3522f50b034b14e41cf62236126, and 51836bf213029b7f0245f8331fa1a2fcc6ff7dda.
